src/util/config_file.cpp

```diff
@@ -0,0 +1,144 @@
+#include "config_file.h"
+
+#include <boost/program_options/option.hpp>
+#include <string>
+#include <set>
+#include <fmt/core.h>
+
+#include "lib/toml.hpp"
+
+// Define this because we cant use log_debug due to this being too early in init.
```

Comment:
I think not because `parseoptions_baset::set_verbosity_msg` is called in `esbmc_parseoptionst::doit`, `cmdlinet::parse` is called in `parseoptions_baset::parseoptions_baset` which is the constructor of the class. So using `log_debug` does not print anything even if `--verbosity 8` is set. Please correct me if I misunderstand.

Reply:
I've spoken to @rafaelsamenezes, and he suggested an alternative, instead I can write some unit tests for the config loader to ensure that it functions as intended. So only error messages will be kept, log messages have been removed.

Reply:
I've written unit tests and removed this :)